lib/facility_factory.rb

```diff
@@ -0,0 +1,35 @@
+class FacilityFactory
+
+  def initialize; end
+
```

Comment:
These methods are a great start. What if we needed to get all 50 states? This class would get pretty large. 
So we would want to make a single `create_facilities(data)` method that could be dynamic in taking whatever data we pass it.
Since each data source designates their keys a little differently, you would probably need to create helper methods that format things like name, address, phone number depending on what source it is coming from. Then you could call those helpers inside your `create_facilities` method.

Reply:
That makes sense, I heard stories around the watering hole that people were doing that and I think at this point I was really proud of the work I had done and also just wanted to be done. I'm glad you said something about this because i'm not so proud of leaving this the way that I left it.

Reply:
Justin, this is a good start. Remember the Red, Green, Refactor. So you absolutely should be proud of what you have done AND understand the potential limitations of things as they are AND ways in which you could refactor to better handle future situations. This is all part of it. You're doing great.